sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -4685,9 +4648,9 @@ public enum AppServicePlanRestriction
         Standard = 4,
         Premium = 5,
     }
-    public partial class AppServicePrivateLinkResource : Azure.ResourceManager.Models.ResourceData
+    public partial class AppServicePrivateLink : Azure.ResourceManager.Models.ResourceData
```

Comment:
We should rename this to `AppServicePrivateLinkResourceData` to be prepared for future changes